gnuradio-runtime/lib/block_executor.cc

```diff
@@ -76,8 +76,9 @@ namespace gr {
       gr::thread::scoped_lock guard(*d->output(i)->mutex());
       int avail_n = round_down(d->output(i)->space_available(), output_multiple);
       int best_n = round_down(d->output(i)->bufsize()/2, output_multiple);
-      if(best_n < min_noutput_items)
+      if(best_n < min_noutput_items) {
```

Comment:
Because this is not a class function and `d_logger` and `d_debug_logger` are not directly accessible. 
I could have used loggers of `block_detail` parameter but then I need to configure get methods of the loggers in `block_detail` class.